lib/passkit/generator.rb

```diff
@@ -99,6 +93,8 @@ def generate_json_pass
         backFields: @pass.back_fields
       }
 
+      pass[:boardingPass ].merge(@pass.boarding_pass) if @pass.pass_type == :boardingPass && @pass.boarding_pass
```

Comment:
```suggestion
      pass[:boardingPass].merge(@pass.boarding_pass) if @pass.pass_type == :boardingPass && @pass.boarding_pass
```

I am not sure that this has any effect. you call merge, but possibly you wanted to do merge! ?